mkdocs.yml

```diff
@@ -156,6 +156,7 @@ nav:
           - Tanium: xdr/features/collect/integrations/endpoint/tanium.md
           - TEHTRIS EDR: xdr/features/collect/integrations/endpoint/tehtris_edr.md
           - Trend Micro Cloud One / Deep Security: xdr/features/collect/integrations/endpoint/trend_micro_deep_security.md
+          - Ubika WAAP Gateway: xdr/features/collect/integrations/cloud_and_saas/ubika_waap.md
```

Comment:
Could you move this entry in the Cloud and Saas section in the mkdocs tree?

Reply:
Sure. Done